src/lib.rs

```diff
@@ -19,11 +19,11 @@ use actor::prelude::*;
 actor_handlers! { codec::http::OP_HANDLE_REQUEST => hello_world, 
                   codec::core::OP_HEALTH_REQUEST => health }
 
-fn hello_world(_payload: codec::http::Request) -> ReceiveResult {
+fn hello_world(_payload: codec::http::Request) -> HandlerResult<codec::http::Response> {
     Ok(serialize(codec::http::Response::ok())?)
```

Comment:
Serialize is no longer needed, the code in actor 0.7.1 invokes that on behalf of actor devs .

This can just be `Ok(codec:: http:: Response::ok())`